### DISCUSSION TITLE
如何使用 SteamTools 1.8 解锁游戏下载？

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#10] [Question / Answer]
如题

**Comment by pjy612** [comment]
要让 tools 能下载游戏，需要以下几个步骤：
1. 关闭Steam。将密钥和最新的清单放入指定位置。（具体参考 wiki ）
2. 通过steamdb.info，定位游戏所需的depot列表。（**注意只能是已获取到密钥的那些，没有密钥加了也是累赘**）
将列表批量框选后拖拽到 tools 悬浮窗。
或将depotid 存放到一个单独的txt文件中，将文件拖拽到 tools 悬浮窗。
注意：要拖入的depotid列表 要能和清单一一对应，不然会出现很多奇怪的问题。
3. （可选）将 DLCId列表 拖入到 tools 解锁DLC（只是解锁DLC开关，具体是否解锁内容得各的DLC是否有对应的额外内容，有内容的得通过depot下载）
4. （可选）将 appId 拖入到 tools 解锁游戏。
5. 通过 tools 启动 Steam。
6. 如果 有出现游戏，则点击进行下载。
如果没有出现游戏，可以通过 steamdb app介绍页的 Install 按钮触发安装。
也可以打开开始菜单-运行，输入 steam://install/appid 来触发安装。
7. 如果不需要后续更新的话，在游戏下载完毕后，按住Ctrl 重复 步骤2，选彻底删除。之后将不会再触发对应游戏的更新。

**特别说明，如果 dlcid 刚好和 app 中 的 depotid 相同，取消 这个 depot 的更新下载 也可能会导致 取消 解锁这个 dlc 。
这种情况，推荐 使用 第三方DLC解锁插件 如 creamapi**

**Reply by genzj** [reply]
请问一下关于“最新的清单”，如果游戏已经迭代清单已经更新，但是还没有人分享过更新后的清单，这种情况下就无法解锁下载了，对吗？即便是想解锁下载更新前的老版本也不可行？

**Reply by pjy612** [reply]
三个方案  
* 使用 steamtools 自行构建 lua 解锁插件。固定 清单版本，可以实现 下载和解锁特定版本。
* 使用 steamtools 在已有游戏文件并卡更新的情况下，直接开启 禁止游戏更新 开关，完全关闭下载功能，跳过后续所有更新和下载。（但是本地已有的非联网游戏可玩）
* 使用 魔改的 DepotDownload ，指定 depot 和 清单id 下载特定版本的游戏资源。

**Reply by genzj** [reply]
非常感谢，我再研究一下
